src/Shellbar/Shellbar.js

```diff
@@ -469,7 +471,10 @@ Shellbar.propTypes = {
     logo: PropTypes.object,
     logoSAP: PropTypes.bool,
     notifications: PropTypes.object,
-    productMenu: PropTypes.array,
+    productMenu: PropTypes.shape({
+        items: PropTypes.array,
+        placement: PropTypes.oneOf(POPPER_PLACEMENTS)
```

Comment:
As an alternative idea to having these be an object with `{items, placement}`, what about adding new props, like this:
```js
productMenuPlacement: PropTypes.oneOf(POPPER_PLACEMENTS),
profileMenuPlacement: PropTypes.oneOf(POPPER_PLACEMENTS)
```
Then it's not a breaking change and it seems more straightforward